### PR TITLE
Fix up CONTRIBUTING

### LINKED ISSUE #133: Add CONTRIBUTING.md
Things to mention:
- [ ]  `&nbsp;` in html/templates
- [ ] jshint
- [ ] dev channel in hipchat
- [ ] scss and js style
- [ ] newline at end of file
- [ ] possible credit with modules like CP3108A/B for more ambitious undertakings

### PR DESCRIPTION
**Pull request by yangshun**
Fixes #133

### CODE REVIEW

File: www/README.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ Mobile browsers:
 
 ## Contributing
 
-Don't know where to start? We highly recommend reading [this fantastic beginner's guide][zames-guide] written by one of our contributors. Alternatively, have a poke at our [open issues](https://github.com/nusmodifications/nusmods/issues?q=is%3Aissue+is%3Aopen).
+Don't know where to start? Firstly read our repository [contribution guide](../CONTRIBUTING.md). Next, we highly recommend reading [this fantastic beginner's guide][zames-guide] written by one of our contributors. Alternatively, have a poke at our [open issues](https://github.com/nusmodifications/nusmods/issues?q=is%3Aissue+is%3Aopen).
```
Comment (taneliang) [inline www/README.md]: /nit s/Firstly/First,

Comment (taneliang) [reply www/README.md]: Also, now that we have quite a few good for beginners issues, maybe we can link to those instead?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
 # Contributing
 
-TBD
+Inspired/copied shamelessly from React's [contribution guide](https://reactjs.org/docs/how-to-contribute.html).
```
Comment (taneliang) [inline CONTRIBUTING.md]: /nit s/Inspired/Inspired by

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+If somebody claims an issue but doesn't follow up for more than two weeks, it's fine to take over it but you should still leave a comment.
+
+## Sending a Pull Request
+
+The core team is monitoring for pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation.
+
+Before submitting a pull request, please make sure the following is done:
+
+1. [Fork the repository](https://github.com/nusmodifications/nusmods) and create your branch from `master`.
+1. Run `yarn` in the root of the project that you are working on.
```
Comment (taneliang) [inline CONTRIBUTING.md]: Maybe this will be clearer if we add "i.e. `/www`, `/export`, `/api` or `/packages/nusmoderator`"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+1. If you've fixed a bug or added code that should be tested, add tests! We ask that you write tests if your feature contains non-trivial logic. Tests can be omitted for minor layout or stylistic changes (or simply generate a snapshot).
+1. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch` is helpful in development.
```
Comment (taneliang) [inline CONTRIBUTING.md]: Our `yarn test:watch` script is better haha. Our `yarn test` scripts generate coverage

File: README.md [hunk 5]
```diff
@@ -15,15 +15,17 @@ This monorepo consists of 4 projects. Further details can be found at the links
...
+* Facebook: https://www.facebook.com/nusmods
 * Twitter: https://twitter.com/nusmods
 * Email: nusmods@googlegroups.com
 
 ## Code of Conduct
 
-We has adopted the same Code of Conduct as Facebook that we expect project participants to adhere to. Please read [the full text](https://code.facebook.com/codeofconduct) so that you can understand what actions will and will not be tolerated.
+We have adopted the same Code of Conduct as Facebook that we expect project participants to adhere to. Please read [the full text](https://code.facebook.com/codeofconduct) so that you can understand what actions will and will not be tolerated.
 
 ## Good First Issues
```
Comment (ZhangYiJiang) [inline README.md]: Can we link `contributing.md` directly from the root README?

Comment (yangshun) [reply README.md]: I don't understand. Anyway if it's a trivial fix, feel free to take over and patch and merge. The timezone differences slow things done a lot.

**Pull request merged**